addons/test_event_full/tests/__init__.py

```diff
@@ -7,4 +7,5 @@
 from . import test_event_mail
 from . import test_event_security
 from . import test_performance
+from . import test_website_event
```

Comment:
From what I see, I would rename the file `test_wevent_menu` (seems to better represent what's tested)
Don't forget to adapt the class name accordingly :slightly_smiling_face: